frontend/packages/console-dynamic-plugin-sdk/src/extensions/resource-metadata.ts

```diff
@@ -13,14 +13,18 @@ export type ModelMetadata = ExtensionDeclaration<
     color?: string;
     /** Override the label. Requires `kind` be provided. */
     label?: string;
+    /** Override plural. Requires `kind` be provided. */
+    plural?: string;
```

Comment:
Isn't the `plural` value part of the CRD definition? 
Why would you ever have a need to override the this value?